tests/fixtures/issues/issue-1360.php

```diff
@@ -1,278 +1,13 @@
 <?php
 
 return [
-    'issue-1360-A' => [
-        'ua' => 'Mozilla/5.0 (Windows; U; WinNT4.0; bg; rv:1.8.1.4) Gecko/20100401 Firefox/4.0',
```

Comment:
A number of agents here (that I added earlier, heh) that are bad platform/browser version combinations and have been moved to the invalid-versions fixture.